lib/spack/spack/build_systems/python.py

```diff
@@ -284,76 +249,124 @@ def test(self):
                 work_dir="spack-test",
             )
 
-    run_after("install")(PackageBase._run_default_install_time_test_callbacks)
 
-    # Check that self.prefix is there after installation
-    run_after("install")(PackageBase.sanity_check_prefix)
+@spack.builder.builder("python_pip")
+class PythonPipBuilder(BaseBuilder):
```

Comment:
Wondering if we should just call this `PipBuilder`. I think pip implies Python.

Reply:
Let me know, this wouldn't be an issue.